app/controllers/users_controller.rb

```diff
@@ -105,6 +105,7 @@ def profile
       redirect_to "/profile/#{current_user.username}"
     else
       @user = DrupalUser.find_by(name: params[:id])
+      @noteGraph = @user.user.daily_note_tally.to_a.sort
```

Comment:
I actually slightly prefer using this inline rather than managing this in the controller; it's concise enough to be readable inline and we don't have to complicate the controller here. If you don't mind, could we do that rather than having a distinct instance variable? Thank you!